src/pkg/projects/depproj.targets

```diff
@@ -0,0 +1,109 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--
+    Shared targets specific to dependency projects (depproj).
+  -->
+
+  <!-- Fetches all the file items from the packages that we want to redist -->
+  <Target Name="GetFilesFromPackages">
+    <ItemGroup Condition="'$(NuGetRuntimeIdentifier)' != ''">
+      <!-- RID-specific: include all runtime files except files in Microsoft.TargetingPack.Private.WinRT -->
+      <FilesToPackage Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NuGetPackageId)' != 'Microsoft.TargetingPack.Private.WinRT'">
```

Comment:
Can you delete this condition?  We should fetch Windows.winmd in a different way so that this isn't needed.

Reply:
I can make the package reference conditional so it's enabled in `src.builds` for crossgen but not `pkg.builds`. `SkipDesignTimePackageReferences` seems like a reasonable thing to call this.

Smells hacky to me, (specifically, restoring the same project multiple times with different references) but it's simple, removes this condition, and at least it isn't unclear in the source what's going on.

Another idea is to fit some project earlier in the build that specifically restores this package and plops the file in an intermediate path for the depproj to use. I suppose this would be similar to the `external/winrt` approach in CoreFX. This would take some more time to implement--I don't see precedent in Core-Setup.

I'll put the condition approach in this PR for now. Moving it out seems like something that can be done later.

Reply:
Cheaper fix would be to make the PackageReference ExcludeAssets=All and then just grab the path from the PackagesDir.  I'm ok with not using nuget resolution for this file.

Reply:
I gave `PrivateAssets=All` a try earlier which didn't seem to affect `PrereleaseResolveNuGetPackageAssets`. I'll try `ExcludeAssets`. Agreed it'd be cheaper.

Reply:
Worked! Thanks.